aiotruenas_client/websockets/protocol.py

```diff
@@ -121,6 +118,17 @@ async def subscribe(self, name: str) -> asyncio.Queue:
         )
         return await sub_future
 
+    @abstractmethod
+    async def _authenticate(self):
```

Comment:
I missed this before: can we add typing here?  I believe it's `Any` given that we're just calling `invoke_method`